qlib/rl/order_execution/simulator_qlib.py

```diff
@@ -23,30 +24,42 @@ class SingleAssetOrderExecution(Simulator[Order, SAOEState, float]):
     ----------
     order
         The seed to start an SAOE simulator is an order.
-    strategy_config
-        Strategy configuration
     executor_config
         Executor configuration
     exchange_config
         Exchange configuration
     qlib_config
         Configuration used to initialize Qlib. If it is None, Qlib will not be initialized.
+    cash_limit:
+        Cash limit.
+    backtest_mode
```

Comment:
`backtest_mode`  is not a necessary parameter if we carefully design it.
It should disappear with `CollectDataEnvWrapper` in the future.

Please add doc for it.

Reply:
Doc added.

Reply:
Agree. `backtest_mode` looks ugly in the init signature.